Looks like it can't find the repo in sonatype?

`[ERROR] Failed to execute goal on project dendrite: Could not resolve dependencies for project org.lab41:dendrite:war:0.1.0.BUILD-SNAPSHOT: Could not find artifact com.atlassian.maven.plugins:bash-maven-plugin:jar:1.0-SNAPSHOT in sonatype-snapshots (https://oss.sonatype.org/content/repositories/snapshots/)`

--- a/pom.xml
+++ b/pom.xml
@@ -468,12 +468,6 @@
             <artifactId>blueprints-graph-jung</artifactId>
             <version>2.4.0</version>
         </dependency>
-        <dependency>
-            <groupId>com.atlassian.maven.plugins</groupId>
-            <artifactId>bash-maven-plugin</artifactId>
-            <version>1.0-SNAPSHOT</version>
-            <scope>provided</scope>
-        </dependency>
     </dependencies>
     <build>
         <plugins>

